composer.json

```diff
@@ -19,7 +19,16 @@
     "autoload": {
         "psr-4": {
             "PaypalPPBTlib\\": "vendor/ppbtlib/src/",
-            "PaypalAddons\\": "."
+            "PaypalAddons\\": ".",
+            "PaypalAddons\\PrestaShop\\PsAccountsInstaller\\": "vendor/prefixed/prestashop/prestashop-accounts-installer/src",
+            "PaypalAddons\\Prestashop\\ModuleLibGuzzleAdapter\\": "vendor/prefixed/prestashop/module-lib-guzzle-adapter/src",
+            "PaypalAddons\\Prestashop\\ModuleLibMboInstaller\\": "vendor/prefixed/prestashop/module-lib-mbo-installer/src",
+            "PaypalAddons\\Clue\\StreamFilter\\": "vendor/prefixed/clue/stream-filter/src",
+            "PaypalAddons\\GuzzleHttp\\Psr7\\": "vendor/prefixed/guzzlehttp/psr7/src",
```

Comment:
Guzzzle is needed ? We made a hard work to make only CURL dependencies, are we sure we want to restore it ?

Reply:
Guzzle is one of dependencies of prestashop/module-lib-guzzle-adapter